Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -381,6 +381,12 @@ out _analysisProcess
                     }
                     entry.SearchPathEntry = childFile.parent;
                     break;
+                case AP.UnhandledExceptionEvent.Name:
+                    Debug.Fail("Unhandled exception from analyzer");
+                    var exception = (AP.UnhandledExceptionEvent)e.Event;
+
+                    VisualStudio.Shell.ActivityLog.LogError(Strings.ProductTitle, exception.message);
```

Comment:
The problem is it's based around taken an exceptions and tasks and not a string object, and we had to turn the exception into a string to send it across the wire.

---
In reply to: [63613169](https://github.com/Microsoft/PTVS/pull/1299#discussion_r63613169) [](ancestors = 63613169)